scanpipe/pipelines/deploy_to_develop.py

```diff
@@ -197,6 +198,25 @@ def flag_mapped_resources_archives_and_ignored_directories(self):
         flag.flag_ignored_directories(self.project)
         d2d.flag_processed_archives(self.project)
 
+    def perform_house_keeping_tasks(self):
+        """
+        On deployed side
+            - PurlDB match files with ``no-java-source``, ``too-many-maps`` and empty
+                status, if no match is found update status to ``requires-review``.
+            - Update status for uninteresting files.
+
+        On devel side
+            - Update status for not deployed files.
+        """
+        d2d.match_resources_with_no_java_source(project=self.project, logger=self.log)
+        d2d.match_unmapped_resources(
+            project=self.project,
+            matched_extensions=self.purldb_resource_extensions,
+            logger=self.log,
+        )
+        d2d.flag_undeployed_resources(project=self.project)
+        d2d.scan_unmapped_to_files(project=self.project, logger=self.log)
```

Comment:
The scanning part would probably make more sense as its own step, not so much as a housekeeping one.

Reply:
ack